src/neynar-api/hub-grpc.ts

```diff
@@ -0,0 +1,24 @@
+import * as grpc from '@grpc/grpc-js';
+
+export function createApiKeyInterceptor(apiKey: string) {
```

Comment:
@BlinkyStitt Can we attach this function as a method to the class `NeynarAPIClient` in `src/neynar-api/neynar-api-client.ts`  so devs don't have to pass in API key while calling this method.

Reply:
Its not part of Neynar API Client though. Its attached to the hub as an interceptor

Reply:
Then we can have a `HubClient` Class that takes in an API, and this method can be attached to that class. So in future if there is anything related to hub client we can attach it to this client